demos/python_demos/common/images_capture.py

```diff
@@ -0,0 +1,157 @@
+import os
+import sys
+import copy
+
+import cv2
+
+
+class InvalidInput(Exception):
+    message = ''
+
+    def __init__(self, message):
+        InvalidInput.message = message
+
+
+class OpenError(Exception):
+    message = ''
+
+    def __init__(self, message):
+        OpenError.message = message
```

Comment:
Here and in `InvalidInput` `message` should be instance attribute, but not class attribute.

Reply:
Done